### PR TITLE
add solo.io as partner on backstage.io

### PR DESCRIPTION
**Pull request by linsun**
Hey, I could not find guideline on adding a company as a commercial partner (listed in https://backstage.io/community) so I went ahead open a PR to get guidance. (Apologies if this is the wrong way.) Please advise if this is okay for merge or anything else we need to do. We are interested in contributing to backstage and would like to have our company listed as a commercial partner - plan to announce something very soon in the next few weeks.

We are excited about backstage and already have a Solo.io Gloo Platform Portal plugin listed at https://backstage.io/plugins/.

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by linsun** [comment]
Chatted with @Rugvip offline (thank you so much!), and I can circle back to update the PR with the link pointing to the offering once it is avail.

For reference, this is for point 4 in the [doc](https://github.com/cncf/foundation/blob/main/website-guidelines.md).

File: microsite/src/pages/community/index.tsx [hunk 1]
```diff
@@ -81,6 +81,11 @@ const Community = () => {
       url: 'https://roadie.io/',
       logo: 'img/partner-logo-roadie.png',
     },
+    {
+      name: 'solo.io',
+      url: 'https://solo.io/',
```
Comment (linsun) [inline microsite/src/pages/community/index.tsx:86]:
```suggestion
      url: 'https://www.solo.io/spotlight/',
```

**Comment by linsun** [comment]
Hi @Rugvip I have added the url, could you PTAL to see if this is okay to approve?

**Comment by linsun** [comment]
cc @freben too

**Review by benjdlambert — APPROVED** [review]
Thanks :tada: Welcome! Exciting stuff!

**Comment by linsun** [comment]
Thanks so much @benjdlambert - we are so excited to be part of the partner community!!

**Pull request merged**